src/systems/force_torque/ForceTorque.cc

```diff
@@ -84,13 +93,25 @@ class gz::sim::systems::ForceTorquePrivate
   public: Entity GetLinkFromScopedName(const EntityComponentManager &_ecm,
                                        const std::string &_name,
                                        Entity _parentModel) const;
-  /// \brief Create FT sensor
-  /// \param[in] _ecm Mutable reference to ECM.
-  public: void CreateForceTorqueEntities(EntityComponentManager &_ecm);
+
+  /// \brief Create sensor
+  /// \param[in] _ecm Immutable reference to ECM.
+  /// \param[in] _entity Entity of the force-torque sensor
+  /// \param[in] _forceTorque ForceTorqueSensor component.
+  /// \param[in] _parent Parent entity component.
+  public: void AddForceTorque(
+    const EntityComponentManager &_ecm,
+    const Entity _entity,
+    const components::ForceTorque *_forceTorque,
+    const components::ParentEntity *_parent);
+
+  /// \brief Create force-torque sensor
+  /// \param[in] _ecm Immutable reference to ECM.
+  public: void CreateSensors(const EntityComponentManager &_ecm);
 
   /// \brief Update FT sensor data based on physics data
   /// \param[in] _ecm Immutable reference to ECM.
-  public: void Update(const EntityComponentManager &_ecm);
+  public: void UpdateForceTorques(const EntityComponentManager &_ecm);
```

Comment:
On #1281 I tried to reduce the diff between the sensors by making these function names generic, see https://github.com/gazebosim/gz-sim/pull/1281#discussion_r785248236. But since you're already working on #1471 I think this won't stay here for long anyway.

Reply:
I will clean them all up in #1471.

Reply:
Actually, I will fix it here, so that we port back to fortress is cleaner.